index.html

```diff
@@ -13,73 +13,219 @@
+                        <p>During the opening ceremony, we'll go over how to request help from a mentor!</p>
+                    </div>
+                </div>
+                <div class="card">
+                    <button class="accordion-header">
+                        <i class="fa fa-plus fa-plus-orange" aria-hidden="true"></i>
+                        Should I bring my own bricks?
+                    </button>
+                    <div class="panel">
+                        <p>We've got you covered.</p>
                     </div>
                 </div>
             </div>
         </div>
-        <div id="badges">
-            <img src="./assets/bh8/macmade-blkwtbrdrspin.gif" alt="Made with Macintosh badge">
-            <img src="./assets/bh8/adobesvg.gif" alt="This side is SVG powered badge">
-            <img src="./assets/bh8/hackerpowered.gif" alt="Hacker powered badge">
+    </section>
+
+    <!-- Contact -->
+    <section id="contact">
+        <div id="contact-content">
+            <h2>Contact</h2>
+            <p>Feel free to let us know of any questions, comments, and/or concerns at <a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
+            <p>Interested in joining our team? <a href="./club.html">Check out our club</a></p>
+        </div>
+        <div id="contact-socials">
+            <p>Connect with us:</p>
+            <div id="social-icons">
+                <a href="https://twitter.com/brickhackrit" target="_blank">
+                    <i class="fab fa-twitter"></i>
+                </a>
+                <a href="https://www.facebook.com/brickhackrit" target="_blank">
+                    <i class="fab fa-facebook"></i>
+                </a>
+                <a href="https://www.instagram.com/brickhack/" target="_blank">
+                    <i class="fab fa-instagram"></i>
+                </a>
+                <a href="https://snapchat.com/add/brickhack" target="_blank">
+                    <i class="fab fa-snapchat"></i>
+                </a>
+            </div>
         </div>
     </section>
+
+    <!-- Footer -->
     <footer>
-        <p>Interested in sponsoring? <span class="text-break">E-mail <a href="mailto:sponsorship@brickhack.io">sponsorship@brickhack.io</a></span></p>
-        <p>Want to help organize? <span class="text-break"><a href="./club.html">Meet the team</a></span></p>
-        <p>View the <a href="./bh7.html">BrickHack 7 website</a></p>
+        <div id="footer-left">
+            <p><a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
+        </div>
+        <div id="footer-right">
+            <p><a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct</a></p>
+            <p><a href="https://www.rit.edu/academicaffairs/policiesmanual/c070" target="_blank">Privacy Policy</a></p>
+            <p>Copyright &copy; 2021 BrickHack Club</p>
+        </div>
```

Comment:
This might be the underlines messing with my eyes, but it feels like the vertical spacing between elements on mobile is less than the line height (as seen when the copyright wraps), which looks a little funny:
![image](https://user-images.githubusercontent.com/16471311/147011665-f58f0782-b677-4ba9-bbe9-5ce37f0671c7.png)
Could we slightly increase the vertical spacing between elements?

Reply:
fixed line height on all the footer in upcoming fixes commit